pkg/engine/mutation.go

```diff
@@ -182,6 +182,9 @@ func mutateElements(name string, foreach *kyverno.ForEachMutation, ctx *PolicyCo
 
 	patchedResource := resource
 	var allPatches [][]byte
+	if foreach.RawPatchStrategicMerge != nil {
```

Comment:
Do we have the same issue with `mutate.patchStrategicMerge`?

Reply:
Not sure. I will check and update.